rust/parse_ast/src/ast_nodes/import_declaration.rs

```diff
@@ -31,6 +35,20 @@ impl AstConverter<'_> {
       &import_declaration.with,
       end_position + IMPORT_DECLARATION_ATTRIBUTES_OFFSET,
     );
+    // phase
+    let phase_position = end_position + IMPORT_DECLARATION_PHASE_OFFSET;
+    match &import_declaration.phase {
+      swc_ecma_ast::ImportPhase::Source => {
+        self.buffer[phase_position..phase_position + 4].copy_from_slice(&STRING_SOURCE);
+      }
+      swc_ecma_ast::ImportPhase::Defer => {
+        self.buffer[phase_position..phase_position + 4].copy_from_slice(&STRING_DEFER);
+      }
+      swc_ecma_ast::ImportPhase::Evaluation => {
+        // Normal imports use 'instance' phase
+        self.buffer[phase_position..phase_position + 4].copy_from_slice(&STRING_INSTANCE);
```

Comment:
Why is this named `instance` while the spec and SWC names it `evaluation`?
https://arai-a.github.io/ecma262-compare/?pr=3492